current/toolkit-walkthrough.md

```diff
@@ -262,7 +287,7 @@ This will take a fair amount of time, even on a small amount of data. It is very
 When it is done, in the /data file you will have results. The first line should look like:
```

Comment:
When it is done, in the /data file you will have results. -->

When it is done, the results will appear in the data file.

Reply:
Yeah maybe `/data folder` like above